src/main/python/ayab/engine/communication.py

```diff
@@ -118,15 +118,13 @@ def req_test_API6(self):
         self.__ser.write(data)
 
     def req_start_API6(self, start_needle, stop_needle,
-                       continuous_reporting, disable_hardware_beep):
```

Comment:
Why did you remove the disable_hardware_beep setting throughout this PR?
The API.md still shows it as a valid parameter (https://github.com/AllYarnsAreBeautiful/fbs/pull/6).